fair_mango/metrics/metrics.py

```diff
@@ -480,10 +505,11 @@ def ratio(
             result[key] = result_i / result_j
```

Comment:
you can divide by zero with numpy. It s gonna give you "inf":

```python
a = np.array(0)
b = np.array(1)
b / a
>>> inf
```

Reply:
Yup. I removed the `zero_division` parameter completely.